### ISSUE TITLE
tries to commit when when no changes and fails

### ISSUE THREAD

**Issue by FredCoen** [wearerequired/lint-action#469]
added this action to our CI however it fails as it tries to commit no changes.

error output:

"Your branch is up to date with 'origin/Implement-first-full-deposit-flow-on-short-term-bond-issuance-product-#7'.\n" +
    '\n' +
    'nothing to commit, working tree clean\n'

This a known issue? thanks

**Comment by ocean90** [comment]
Can you check issue #140 if some of the mentioned workarounds are working for you too?

**Comment by FredCoen** [comment]
yes the workaround worked. seems lock files changes by linter are preceived as changes but not by git
